### PR TITLE
Remove win98 token

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md [hunk 1]
```diff
@@ -21,15 +21,17 @@ helpviewer_keywords:
...
-  
+
+Windows Forms applications are entirely Unicode-enabled, meaning that each character is represented by a unique number, no matter what the platform, program, or language. For more information about Unicode, see the [Unicode consortium Web site](https://www.unicode.org).
+
+## Benefits of Unicode
+
+The benefits of Unicode-enabled forms include the ability to work with scripts that are Unicode-only, such as Hindi. In addition, you can use multiple languages on a single form. In Unicode, all characters are two bytes long, so no special effort is needed to represent double-byte characters. You can also write a single set of code that will work on all platforms. This is a change from previous versions of Visual Basic, in which you had to write different code for different platforms, such as Windows NT and Windows 98.
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md]:
```suggestion
The benefits of Unicode-enabled forms include the ability to work with scripts that are Unicode-only, such as Hindi. In addition, you can use multiple languages on a single form. In Unicode, all characters are two bytes long, so no special effort is needed to represent double-byte characters. You can also write a single set of code that will work on all platforms. 
```

**Review by Thraka — APPROVED** [review]
Thanks!

File: docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md [hunk 2]
```diff
@@ -21,15 +21,17 @@ helpviewer_keywords:
...
+
+However, certain controls do not support Unicode in Windows 98 and Windows Millennium Edition. These controls, all of which inherit from the common control, will process data with the Windows code pages, as ANSI. These controls are: <xref:System.Windows.Forms.TabControl>, <xref:System.Windows.Forms.ListView>, <xref:System.Windows.Forms.TreeView>, <xref:System.Windows.Forms.DateTimePicker>, <xref:System.Windows.Forms.MonthCalendar>, <xref:System.Windows.Forms.TrackBar>, <xref:System.Windows.Forms.ProgressBar>, <xref:System.Windows.Forms.ImageList>, <xref:System.Windows.Forms.ToolBar>, and <xref:System.Windows.Forms.StatusBar>. As a result, you cannot display Unicode data in these controls on the listed platforms. For example, you cannot display Japanese characters on an English Windows 98 operating system.
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md]:
```suggestion
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for removing this little-used token, @Youssef1313. I've suggested some changes to address the fact that this content is extremely dated.

**Comment by rpetrusha** [comment]
Sorry, @Youssef1313. I'll make the suggested changes in a separate PR.

**Comment by Thraka** [comment]
@rpetrusha Sorry about that!! I had this open from earlier and GitHub didn't automatically update that you had done a review.

**Pull request merged**